src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -171,6 +173,47 @@ class ViewFeesFines extends React.Component {
     />;
   }
 
+  showComments = (feeFine) => {
+    const feeFineType = feeFine.feeFineType ? feeFine.feeFineType : '';
+    const comments = _.get(this.props.resources, ['comments', 'records'], []);
+    const actions = _.orderBy(comments.filter(c => c.accountId === feeFine.id), ['dateAction'], ['asc']);
+    const myComments = actions.filter(a => a.comments).map(a => a.comments);
+    const commentsAmount = myComments.length;
+
+    return (
+      <div>
+        <Row>
+          <Col>{feeFineType}</Col>
+          {(commentsAmount > 0) ?
+            <Col style={{ marginLeft: '5px' }}>
+              <Popover>
+                <div data-role="target">
+                  <Icon
+                    icon="comment"
+                    size="small"
+                  />
+                </div>
+                <p data-role="popover">
+                  <b>
+                    <FormattedMessage id="ui-users.accounts.history.staff.info" />
+                    {` ${commentsAmount} `}
+                    <FormattedMessage id="ui-users.accounts.history.of" />
+                    {` ${commentsAmount}`}
+                    :
+                  </b>
```

Comment:
This would be better handled as a single string:
```
// ./ViewFeesFines.js
<FormattedMessage 
  id="ui-users.accounts.history.staffInfoCount" 
  values={{ count: commentsAmount }}
/>

// /translations/ui-users/en.json
"accounts.history.staffInfoCount": "Staff info {count, number} of {count, number}:", 
```